File: devskill/DCP-13.c
#include<stdio.h>

int main(){

    int t,i,j;
    scanf("%d",&t);

    for(i=0;i<t;i++){
        int win=0,ban1,ban2,pak1,pak2,i,j;

        for(j=0;j<2;j++){
            scanf("%d %d %d %d",&ban1,&ban2,&pak1,&pak2);
            if((ban1+ban2)>(pak1+pak2))
                win++;
        }
        if(win==2){
            printf("Banglawash\n");

        }else{
            printf("Miss\n");
        }

    }
return 0;
}

File: devskill/DCP-13.py
T = int(input())
i=1
while T-=1:
	ban1 = int(input())
	ban2 = int(input())
	pak1 = int(input())
	pak2 = int(input())

	if ban1+ban2>pak1+pak2:
		i+=1
	elif i==2:
		print("Banglawash")
	else:
		print("Miss")